### PR TITLE
feat(message): migrate component message - FRONT-1767

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

migrate component message to new structure

### CODE REVIEW

File: src/implementations/twig/components/message/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- **"variant"** (string) (default: ''): could be 'info, 'success', 'warning' or 'error'
+- **"icon"** (object of type "icon") (default: {})
+- **"title"** (string) (default: '')
+- **"close"** (object of type "button") (default: {}): close button
+- **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
+- **"extra_attributes"** (optional) (array) (default: []) Extra attributes
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+- **"_compliance_"** (optional) (boolean) (default: false) Activates debug
```
Comment (planctus) [inline src/implementations/twig/components/message/README.md]:
```suggestion
- **"\_compliance\_"** (optional) (boolean) (default: false) Activates debug
```

File: src/implementations/twig/components/message/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- **"_compliance_inner_check_"** (optional) (boolean) (default: false) Inline compliance report
```
Comment (planctus) [inline src/implementations/twig/components/message/README.md]:
```suggestion
- **"\_compliance_inner_check\_"** (optional) (boolean) (default: false) Inline compliance report
```

File: src/implementations/vanilla/components/message/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Message
```
Comment (planctus) [inline src/implementations/vanilla/components/message/README.md]: I've never fully understood the reason why, but we write this in "plural".. :)
```suggestion
# Messages
```

File: src/implementations/vanilla/components/message/message-print.scss [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+ * Message
+ * @define message
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/spacing';
+@use '@ecl/theme-dev/maps/typography';
+
+$_boder-width: 0.5mm;
```
Comment (planctus) [inline src/implementations/vanilla/components/message/message-print.scss]: border, no..? :)

File: src/implementations/vanilla/components/message/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "3.0.0",
+  "description": "ECL Message",
+  "main": "message.js",
+  "module": "message.js",
+  "style": "message.scss",
+  "sass": "message.scss",
+  "dependencies": {
+    "@ecl/dom-utils": "^3.0.0",
```
Comment (planctus) [inline src/implementations/vanilla/components/message/package.json:12]: I noticed that i forgot to add this dependency in the vanilla package of the file, would you add it in this pr..?

File: src/website/src/pages/ec/components/message/docs/usage.mdx [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph, Anatomy, Do, Caution } from '@ecl/website-components';
```
Comment (planctus) [inline src/website/src/pages/ec/components/message/docs/usage.mdx]: Do we need this Do and Caution..?

**Review by planctus — CHANGES_REQUESTED** [review]
very minor changes.. :)

File: src/implementations/vanilla/components/message/package.json (hunk 5 above)
Comment (kalinchernev) [reply src/implementations/vanilla/components/message/package.json:12]: I've added all missing in #1744
Well-noticed for future packages depending on the queryOne

File: src/implementations/twig/components/message/README.md (hunk 2 above)
Comment (kalinchernev) [reply src/implementations/twig/components/message/README.md]: I notice these changing back and forth: what's the deal with this?

Comment (emeryro) [reply src/implementations/twig/components/message/README.md]: I guess that the "correct name" of the parameter is `_compliance_inner_check_` (with extra underscores), but in markdown it is interpreted as italic if we don't escape it

**Pull request merged**